src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    sex,
+    age,
+    partnerName,
+    isMarried,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age !== undefined && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      {isMarried
+        ? (
+          <p className="Person__partner">
+            {`${partnerName} is my ${sex === 'f'
+              ? 'husband'
+              : 'wife'}`}
+          </p>
+        )
+        : (
+          <p className="Person__partner">I am not married</p>
```

Comment:
firstly you can do this check inside `p` tag, so you will not write it twice and write less code

secondly, create a variable for this check:
`const partner = sex === 'f' ? 'husband'  : 'wife'`

Reply:
thx that looking better 😋